migrations/1457249988959-add-roles.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// load .env
+require('dotenv').load();
+
+const Roles = require('../server/models/roles');
+const titles = Roles.schema.paths.title.enumValues;
+
+exports.up = function(next) {
+
+  let tasks = titles.map((title) => {
+    let query = {title: title};
+    let update = {
+      title: title
+      //accessLevel: 0 // FIXME: Is there a default accessLevel?
```

Comment:
If `user: 0` is the same as role `viewer` then we could do away with `accessLevel` altogether and use role alone. Then we could remove the role model and change the `role` fields in `UserSchema` and `DocumentSchema` to a string with enum `[viewer, staff, admin]`.